lib/primitive/lib/Cardano/Wallet/Primitive/Types/Tx/Constraints.hs

```diff
@@ -23,8 +23,6 @@ module Cardano.Wallet.Primitive.Types.Tx.Constraints
     , txOutputHasValidTokenQuantities
     , TxSize (..)
     , txSizeDistance
-    , TokenBundleSizeAssessor (..)
-    , TokenBundleSizeAssessment (..)
```

Comment:
Are these different enough from the rest of the `Tx.Constraints` module to warrant the separation? I guess in that `TxConstraints` is used for migration...

Reply:
AFAIK the rest of the `TxConstraints` module is not (currently) used by `cardano-coin-selection` library. As you point out, it was created for the purpose of the migration algorithm.

My goal here is really to isolate that which is essential for `cardano-coin-selection`.